lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -189,6 +189,7 @@ export default function ContentSelector({
           onSelectFile={selectD2LFile}
           missingFilesHelpLink={'https://web.hypothes.is/help-categories/d2l/'}
           withBreadcrumbs
+          apiSendsChildren
```

Comment:
I initially was checking if "children" was present in the response all the time but I reckon it makes more sense to be more  explicit about it.